spring-cloud-function-adapters/spring-cloud-function-adapter-gcp/src/main/java/org/springframework/cloud/function/adapter/gcloud/GcfSpringBootHttpRequestHandler.java

```diff
@@ -16,38 +16,24 @@
 
 package org.springframework.cloud.function.adapter.gcloud;
 
-import java.io.BufferedReader;
-import java.io.BufferedWriter;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-
-import com.google.cloud.functions.Context;
 import com.google.cloud.functions.HttpFunction;
 import com.google.cloud.functions.HttpRequest;
 import com.google.cloud.functions.HttpResponse;
 import com.google.gson.Gson;
 import org.reactivestreams.Publisher;
-import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
 
 import org.springframework.cloud.function.context.AbstractSpringFunctionAdapterInitializer;
-import org.springframework.messaging.Message;
-import org.springframework.messaging.MessageHeaders;
-import org.springframework.messaging.support.GenericMessage;
 
 /**
- * Implementation of HttpFunction for Google Cloud Function.
- *
- * @param <O> input type
+ * Implementation of {@link HttpFunction} for Google Cloud Function (GCF).
+ * This is the Spring Cloud Function adapter for GCF HTTP function.
```

Comment:
Add javadoc for what is the generic type `O`

Reply:
I don't really need it right now. Just removed.